doc/src/manual/missing.md

```diff
@@ -254,6 +254,23 @@ julia> Array{Union{Missing, String}}(missing, 2, 3)
  missing  missing  missing
 ```
 
+For `T` that is bits type (`isbitstype(T)` returns `true`),
+`Array{Union{Missing, T}}(undef, dims)` creates an array filled with
```

Comment:
Better to recommend `Array{Union{Missing, T}}(undef, dims)` (which should work even for non-`isbits` `T`)?

Reply:
This is the problem. If `T` is non `isbitstype` then the array is truly uninitialized (`#undef`). If I understand the reason correctly this is for performance reasons.

Reply:
You mean `Array{Union{Missing, T}}(missing, dims)` @fredrikekre?

I'm not sure whether it's a good idea to develop this here given that we want people to use `Array{Union{Missing, T}}(missing, dims)` rather than `Array{Union{Missing, T}}(undef, dims)`, and that it makes things more complex with the introduction of the concept of `isbits` type. Maybe make this a shorter `!!note` without showing the REPL output? It would also be useful to mention for non-`isbits` type the entries will be `#undef`.

Reply:
> You mean...

Yea...